tests/run-make/reset-codegen-1/rmake.rs

```diff
@@ -28,11 +28,13 @@ fn main() {
         ("link-output", Some("link")),
         ("obj-output", Some("obj")),
         ("dep-output", Some("dep-info")),
-        ("multi-output", Some("asm,obj")),
+        // ("multi-output", Some("asm,obj")),
+        // FIXME(Oneirical): Comment in the original Makefile:
+        // (This case doesn't work yet, and may be fundamentally wrong-headed anyway.)
```

Comment:
I'll have to take a closer look at this.

Reply:
Problem: I know what the original Makefile test says, but the ported rmake.rs version must've already passed when we merged https://github.com/rust-lang/rust/pull/125227, so we should not comment this test out.